utils/db/relevance.ts

```diff
@@ -21,17 +23,16 @@ export interface DbHunks {
 	}
 }
 
-export const saveHunk = async (hunkInfo: string) => {
-	const hunkinfo_json = JSON.parse(hunkInfo);
-	console.info("[saveHunk] Saving hunk for ", hunkinfo_json.repo_name);
-	for (const prHunk of hunkinfo_json.prhunkvec) {
+export const saveHunk = async (hunkInfo: DPUHunkInfo) => {
+	console.info(`[saveHunk] Saving hunk for ${hunkInfo.repo_provider}/${hunkInfo.repo_owner}/${hunkInfo.repo_name}`);
+	for (const prHunk of hunkInfo.prhunkvec) {
```

Comment:
you removed this parsing of `hunkInfo` into JSON since the function is already given `hunkInfo` as JSON in the `hunkHandler` api right?

Reply:
Earlier, we were passing the hunk info as a string to this function. It had to be parsed within the function.
When I created the new function, the `JSON.parse` was getting repeated. So I took out that common job. 
And in the process, I also made this function better because now it clearly indicates what is a good input right from its signature.